client/components/search/index.jsx

```diff
@@ -24,6 +23,7 @@ import TranslatableString from 'components/translatable/proptype';
  * Internal variables
  */
 const SEARCH_DEBOUNCE_MS = 300;
+let instanceCount = 0;
```

Comment:
I wonder if we could use lodash `uniqueId` here or if we had a reason for us counting the exact number of instances. (lodash just increments a number internally). From usage it currently looks like we just need it for a unique key.

Reply:
That's an excellent idea! I honestly didn't look for alternatives because the same pattern is used in a couple of components. But I've updated the code on c3d70474febff4c36a711d9cec333f9e00b590d3 to use `uniqueId`.

Reply:
I think we're okay with the change, maybe we have some SSR requirements? Any of that sound familiar @ockham? I'll run the full suite again anyway.